### ISSUE TITLE
[BUG] Release APK is set as Debug

### ISSUE THREAD

**Issue by sebastien46** [VadimBoev/FlappyBird#40]
The current version `1.7` APK is set to `android:debuggable=true`.
This is potentially undesirable behavior, as it:
- Opens up security holes: https://developer.android.com/privacy-and-security/risks/android-debuggable
- Significantly affects app performance: https://medium.com/specto/dont-run-benchmarks-on-a-debuggable-android-app-like-i-did-34d95331cabb

The solution would be either to:
- Split the APK into Two Release / Debug APKs
- Set the app to `android:debug=false` and only keep the release APK.

**Comment by VadimBoev** [comment]
Yes, you're definitely right, I'll keep that in mind if there are further releases.
